### ISSUE TITLE
The data support orders of magnitude

### ISSUE THREAD

**Issue by Invincibled** [openlink/virtuoso-opensource#980]
Hello, I would like to ask about the data volume level supported by Virtuoso. Can it support up to TB?
How efficient is it at the TB level? Has it been tested?

Looking forward to your reply. Thank you

**Comment by HughWilliams** [comment]
Yes, Virtuoso can handle multiple TB data volumes as can been see for some of the publicly hosted instances as detailed in spreadsheet [here](https://docs.google.com/spreadsheets/d/1-stlTC_WJmMU3xA_NxA1tSLHw6_sbpjff-5OITtrbFw/edit#gid=812792186) ...

**Comment by ebremer** [comment]
I have 20 billion triples loader in my virtuoso instance.  Runs like a champ.
